automatic/k9s/tools/chocolateyinstall.ps1

```diff
@@ -5,15 +5,15 @@ $toolsDir      = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
 $packageArgs = @{
   PackageName    = $packageName
-  FileFullPath64 = Get-Item $toolsDir\k9s_*Windows_x86_64.tar.gz
+  FileFullPath64 = Get-Item $toolsDir\k9s_*Windows_*64.tar.gz
```

Comment:
I know this code was pre-existing and you are just updating it. However, we want to remove `Get-Item` when looking for embedded binaries. Please reference the exact name of the embedded file here without using `Get-Item` or similar, to search for it.